packages/jmix-front-generator/src/init.ts

```diff
@@ -132,13 +132,19 @@ export async function generate(
   templateOverride?: string,
 ): Promise<void> {
   const env = new YeomanEnvironment();
+  const defaultOptions = {force: true};
```

Comment:
So this is probably the reason for removing `conflicter: true`?

But we don't always want to force rewrite, we don't want it when we are invoking CLI in an interactive mode.

Reply:
Yes. Conflicter was moved to the Environment.
And now CLI doesn't ask about rewriting.

Reply:
Now in an interactive mode, we ask about rewriting a file